### PR TITLE
Sample code

### PR DESCRIPTION
**Pull request by waterfoul**

### CODE REVIEW

File: GraphQL.Annotations.TSql/ExtensionMethods.cs [hunk 1]
```diff
@@ -20,18 +27,205 @@ public static class ExtensionMethods
...
+		        ));
+	        }
+
+	        return graphType;
+        }
+
+        private static IEnumerable<MethodInfo> GetMutationMethodInfos<T>()
+        {
+	        return typeof(T).Assembly.GetTypes()
+		        .Where(t => t.IsPublic)
```
Comment (AmySchaplowsky) [inline GraphQL.Annotations.TSql/ExtensionMethods.cs]: using a single where clause and && your conditions would be more efficient

File: GraphQL.Annotations.TSql/SqlFieldMutator.cs [hunk 2]
```diff
@@ -41,7 +41,7 @@ public static class SqlFieldMutator
 				.Where(v => v.Attribute.ReturnType != typeof(IdGraphType))
 				.ToList();
 			var notIdentity = fields
-				.Where(v => !v.Attribute.IsIdentityField)
+				.Where(v => !v.Attribute.SkipOnInsert)
```
Comment (AmySchaplowsky) [inline GraphQL.Annotations.TSql/SqlFieldMutator.cs:44]: consolidate where clause

**Review by AmySchaplowsky — CHANGES_REQUESTED** [review]
You have sever Where clauses strung together that can be consolidated into one.  That will help performance.

File: GraphQL.Annotations.TSql/SqlFieldMutator.cs (hunk 2 above)
Comment (waterfoul) [reply GraphQL.Annotations.TSql/SqlFieldMutator.cs:44]: This where clause cannot be consolidated since it is the only one

**Review by AmySchaplowsky — APPROVED** [review]

**Pull request merged**